Makefile

```diff
@@ -1,9 +1,53 @@
+SHELL = /bin/bash
 # We assume an active virtualenv for development
-install:
-	-@python3 setup.py -q develop --upgrade
+include requirements.txt
+PYENV_LOCATION = /usr/local/bin/pyenv
+PYENV_REGEX = .pyenv/shims
+VENV_NAME ?= .venv
+VENV_ACTIVATE_FILE = $(VENV_NAME)/bin/activate
+VENV_ACTIVATE = . $(VENV_ACTIVATE_FILE)
+VEPYTHON = $(VENV_NAME)/bin/python3i
+PYENV_ERROR = "\033[0;31mIMPORTANT\033[0m: Please install pyenv. \033[0;31m\n"
```

Comment:
Remove `\033[0;31m` at the end of the string. Already resetting with `\033[0m` earlier.

Reply:
Fixed.